### PR TITLE
Javascript table sort

### PR DESCRIPTION
**Pull request by toolstack**
Replace the WP_List_Table sorting with client side JavaScript sorting.

### CODE REVIEW

**Comment by MickeyKay** [comment]
@toolstack Why would you want to replace the default functionality that WP provides?

**Comment by toolstack** [comment]
To avoid a page reload.

**Comment by toolstack** [comment]
Also the client side sorting allows for multiple keys to be sorted on instead of just one.

**Comment by MickeyKay** [comment]
This would be cool, however I would rather stick to WP core. That way if
list table sorting is ever updated there, we'll stay compatible with their
changes. I know this would be a nice to have, but my philosophy on this one
is to stick with core.

**Comment by toolstack** [comment]
The other way to do it would be to dump the WP_List_Table code all together and just use the standard classes associated with the WP_List_Table code, which would avoid any incompatibilities in the future.

**Review by MickeyKay — CHANGES_REQUESTED** [review]
So. . . here are my thoughts. I really like the idea of incorporating this FE sorting, however I see no reason to switch away from all of the handiness that WP_List_Table gives us. By manually outputting the table, we lose all of the native functionality and user-accessible filters/actions that are actually pretty useful. My strong preference would be to keep the WP_List_Table output mechanism, then simply unhook WP's core table JS and override with your tablesort JS.

File: admin/js/jquery.tablesorter.js [hunk 1]
```diff
@@ -0,0 +1,1901 @@
+/**!
```
Comment (MickeyKay) [inline admin/js/jquery.tablesorter.js:1]: If we're going to include this file, it should definitely be minified unless `SCRIPT_DEBUG` is set to true.

File: admin/class-wp-dev-dashboard-admin.php [hunk 2]
```diff
@@ -182,6 +183,7 @@ public function enqueue_styles() {
 	public function enqueue_scripts() {
```
Comment (MickeyKay) [inline admin/class-wp-dev-dashboard-admin.php]: I'm realizing we should only be enqueueing these on specific admin screens. Any chance you want to add that conditional?

**Comment by toolstack** [comment]
I rebased the PR and left in the original WP_List_Table code.

File: admin/js/jquery.tablesorter.js (hunk 1 above)
Comment (toolstack) [reply admin/js/jquery.tablesorter.js:1]: Done.

File: admin/class-wp-dev-dashboard-admin.php (hunk 2 above)
Comment (toolstack) [reply admin/class-wp-dev-dashboard-admin.php]: Done.

**Pull request merged**